web/src/features/moreCast2/saveForecasts.ts

```diff
@@ -12,7 +13,7 @@ export const isForecastRowPredicate = (row: MoreCast2Row) =>
   isNaN(row.grassCuringActual)
 
 export const getForecastRows = (rows: MoreCast2Row[]): MoreCast2Row[] => {
-  return rows ? rows.filter(isForecastRowPredicate) : []
+  return rows ? rows.filter(row => isForecastRowPredicate(row) && !isBeforeToday(row.forDate)) : []
```

Comment:
What's the difference between `isForecastRow` and `isForecastRowPredicate`?

Reply:
I looked at replacing `isForecastRowPredicate` with `isForecastRow` but a few things broke and I reverted my change. `isForecastRow` takes into consideration the `forDate` whereas `isForecastRowPredicate` does not. `isForecastRow` calls another function, `rowContainsActual` which only considers a row to be 'actual' if it is completely populated with actual values from WF1, including FWI values. 

I'm going create a dev task to review and simplify some of this logic.